src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -652,6 +659,65 @@ public int getHeight(int x, int z) {
         return heightMap[z * WIDTH + x] & 0xff;
     }
 
+    public float getRegionalDifficulty() {
+        final float moonPhase = world.getMoonPhase();
```

Comment:
Why are these floats and not doubles?

Reply:
At first I thought that using float was the right choice with java math, but now i did some researches and I understand that I should use double for this kind of things. Should I update [this method](https://github.com/GlowstoneMC/Glowstone/pull/1019/files/56fccd7a621490c98c9e881b3917a605093a73cc#diff-cea1d25e5518f2495b11e2d20021b0a0R751) too?

Reply:
Yes please :)